sql_generators/glean_usage/templates/events_first_seen_v1.query.sql

```diff
@@ -0,0 +1,140 @@
+{{ header }} -- the header doesn't seem to work properly
```

Comment:
The header isn't automatically added for `generate_across_apps` so you would need to add it yourself. https://github.com/mozilla/bigquery-etl/blob/69d9173b5d4406b244a4d1cb855ad835e8dfcb4b/sql_generators/glean_usage/common.py#L527-L532

But `generate_across_apps` seems like the wrong choice here since it will only run once, meaning if you added another app to the bqetl_project.yaml, it would only generate a query for one app.  `generate_per_app_id` is more appropriate since events_stream is per app id